play/src/pusher/app.ts

```diff
@@ -22,6 +24,7 @@ import { CompanionService } from "./services/CompanionService";
 import { WokaService } from "./services/WokaService";
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const LiveDirectory = require("live-directory");
+const monitoringInterface: MonitoringInterface = new SentryBack();
```

Comment:
The name of the variable should never contain the "Interface" suffix.

You should probably rename this to `monitoring` or `monitoringService`.

```suggestion
const monitoring: MonitoringInterface = new SentryBack();
```